docs/content/en/docs/installation/runtime-hooks.md

```diff
@@ -89,27 +89,106 @@ Install the new configuration file and restart containerd
 minikube cp /tmp/new-config.toml /etc/containerd/config.toml
 minikube ssh sudo systemctl restart containerd
 ```
+{{% /tab %}}
+
+{{% tab "kind with Containerd (default)" %}}
```

Comment:
I think what you want to convey here is that Containerd running with kind is the default (for kind). It looks a little bit from the tabs that it's the default way of installing runtime hooks (even if that makes little sense). Maybe just drop the (default) and add a small note at the start of the guide that Containerd is the default when deploying kind.

<img width="784" alt="image" src="https://github.com/user-attachments/assets/bef041d9-2cb1-442a-9d3c-474dda3b3f68" />